whispermovie/openai.py

```diff
@@ -95,11 +95,11 @@ def openai_audio(audio_file: BinaryIO, prompt: str,
     request_params = {
         "model": model,
         "prompt": prompt,
-        "response_format": response_format,
+        "response_format": response_format
     }
 
     if language is not None:
-        request_params["language"] = language
+        request_params["lang"] = language
```

Comment:
I believe it's actually langugae according to [OpenAI](https://platform.openai.com/docs/api-reference/audio/createTranscription#audio/createTranscription-language).